templates/download/risc-v/tab-5.html

```diff
@@ -1,47 +1,37 @@
-<div tabindex="0" role="tabpanel" id="live-installer-tab" aria-labelledby="live-installer" aria-hidden="true">
+<div tabindex="0" role="tabpanel" id="microchip-polarfire-soc-fpga-icicle-kit-tab" aria-labelledby="live-installer" aria-hidden="true">
 	<div class="row u-vertically-center u-no-padding">
 		<div class="col-6">
-			<h2>Download Ubuntu Server Live Installer</h2>
-			<h3 class="p-heading--4">Ubuntu Server 22.10</h3>
-			<div class="p-notification--information is-inline">
-				<div class="p-notification__content">
-					<h4 class="p-notification__title">Note:</h4>
-					<p class="p-notification__message">It is an early RISC-V developer access through Ubuntu 22.10. This image works on the SiFive Unmatched Board and on QEMU.</p>
-				</div>
-			</div>
-			<p>
-				<a class="p-button--positive" href="https://cdimage.ubuntu.com/releases/22.10/release/ubuntu-22.10-live-server-riscv64.img.gz">Download 64-bit</a>
-			</p>			
+			<h2>Download Ubuntu Server</h2>
 			<h3 class="p-heading--4">Ubuntu Server 22.04.2</h3>
 			<div class="p-notification--information is-inline">
 				<div class="p-notification__content">
 					<h4 class="p-notification__title">Note:</h4>
-					<p class="p-notification__message">It is an early RISC-V developer access through Ubuntu 22.04.2. This image works on the SiFive Unmatched Board and on QEMU.</p>
+					<p class="p-notification__message">It is an early RISC-V developer access through Ubuntu 22.04.2.</p>
```

Comment:
```suggestion
					<p class="p-notification__message">It is an early RISC-V developer access through Ubuntu 22.04.2.</p>
```
I think are you right in this being 22.04.2, but it is still 22.10 in the doc so I have left a comment for Felicia

Reply:
Felicia confirmed